### ISSUE TITLE
The `@ignore` filter parameter inheritance does not take effect when the request method is GET.

### ISSUE THREAD

**Issue by XDreamc** [TongchengOpenSource/smart-doc#42]
![image](https://user-images.githubusercontent.com/43606154/86199063-af0e0000-bb8b-11ea-874d-2f37c06b658d.png)

![image](https://user-images.githubusercontent.com/43606154/86199276-3bb8be00-bb8c-11ea-9da6-10fee2cbe13a.png)

![image](https://user-images.githubusercontent.com/43606154/86199264-30fe2900-bb8c-11ea-8cbb-324725583c28.png)

I used `@ignore` to filter the request base class, but it didn't work when the request method was GET.

**Comment by XDreamc** [comment]
Controller layer code 
```
 /**
  * Department is exist
  */
@GetMapping("/is_exist")
 public Result isExist(@Valid DepartmentNameCheckReq req) {
        return Result.buildSuccessResult(true);
 }
```

**Comment by shalousun** [comment]
@XDreamc 
There are no issues found in the current test. You can check if the test case is consistent with your method here:
https://github.com/shalousun/api-doc-test/blob/master/src/main/java/com/power/doc/controller/IgnoreTagController.java
If it's not consistent, you can fork this smart-doc test demo, add your reproduction test case, and then submit a pull request.
当前测试未发现问题，你可查看改测试用例时候和你方式一致
https://github.com/shalousun/api-doc-test/blob/master/src/main/java/com/power/doc/controller/IgnoreTagController.java
如果不一致你和fork这个smart-doc的测试demo添加你的复现用例测试后提个pull request

**Comment by XDreamc** [comment]
> @XDreamc There are no issues found in the current test. You can check if the test case is consistent with your method here:
https://github.com/shalousun/api-doc-test/blob/master/src/main/java/com/power/doc/controller/IgnoreTagController.java
If it's not consistent, you can fork this smart-doc test demo, add your reproduction test case, and then submit a pull request.
当前测试未发现问题，你可查看改测试用例时候和你方式一致
https://github.com/shalousun/api-doc-test/blob/master/src/main/java/com/power/doc/controller/IgnoreTagController.java
如果不一致你和fork这个smart-doc的测试demo添加你的复现用例测试后提个pull request

Using your test case, it will not be ignored either.
![image](https://user-images.githubusercontent.com/43606154/86329088-451f5480-bc78-11ea-989c-51da4f664fb8.png)
![image](https://user-images.githubusercontent.com/43606154/86329373-a6dfbe80-bc78-11ea-9ed4-8d6d276f719b.png)
![image](https://user-images.githubusercontent.com/43606154/86329427-bced7f00-bc78-11ea-8846-31ec5a410760.png)
![image](https://user-images.githubusercontent.com/43606154/86329505-d7bff380-bc78-11ea-8e9c-136f450e6afc.png)

**Comment by shalousun** [comment]
@XDreamc Please debug whether smart-doc reads the `ignore` tag in your own environment. We are unable to reproduce it.
![image](https://user-images.githubusercontent.com/9592228/86332215-c678e600-bc7c-11ea-9f86-dabf8ad239f6.png)

**Comment by XDreamc** [comment]
@shalousun 
![image](https://user-images.githubusercontent.com/43606154/86341505-f7f7ae80-bc88-11ea-94b7-88790850cbd9.png)
![image](https://user-images.githubusercontent.com/43606154/86342062-ac91d000-bc89-11ea-9697-35b917762a61.png)
![image](https://user-images.githubusercontent.com/43606154/86342450-35107080-bc8a-11ea-88a9-8278a5d08547.png)
![image](https://user-images.githubusercontent.com/43606154/86342855-bf58d480-bc8a-11ea-8512-5cd4a8d0012b.png)
![image](https://user-images.githubusercontent.com/43606154/86342947-e0212a00-bc8a-11ea-9eb1-213dd806ea42.png)
When executing up to this point, it has already gone wrong. Therefore, the output results did not remove the parameters from the GET request.

**Comment by XDreamc** [comment]
In the FormDataBuildHelper code, I did not see any logic to ignore parameters as follows.
![image](https://user-images.githubusercontent.com/43606154/86348972-064ac800-bc93-11ea-91ed-11a0d01d9999.png)
@shalousun  Please check why the output from Postman JSON during your tests is inconsistent with the code logic. I don't understand why.

**Comment by shalousun** [comment]
@XDreamc That's the latest code we fixed. The previous version was missed in the release. You can pull the dev branch code and compile it for verification first.

**Comment by XDreamc** [comment]
@shalousun  Yes, I see that you fixed it after 1.8.8. It seems you need to release 1.8.9 now.
